UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -48,7 +48,7 @@ public abstract class LivingHealthMasterBase : NetworkBehaviour, IFireExposable,
 		/// <summary>
 		/// Rate at which periodic damage, such as radiation, should be applied
 		/// </summary>
-		private float tickRate = 1f;
+		[SerializeField] private float tickRate = 1f;
```

Comment:
oh?

Reply:
why is this opened to the inspector?

Reply:
mobs don't need to be as actively checked as players

Reply:
might be better to do it if a player is not possessing it?

Reply:
or you can just expose the field and leave it as is 
players possessing other NPCs aren't high on the list right now to warrent adding a way to change the tickrate based on if a player is possessing or not.

it also makes the behavior much more consistent regardless of what is controlling the mob

Reply:
but gives inconsistent behaviour if you put a body part from a mob into a player or vice versa , I'm fine if you  just slap a TODO on it